fdbrpc/Locality.h

```diff
@@ -24,6 +24,9 @@
 
 #include "flow/flow.h"
 
+inline const StringRef ExcludeLocalityKeyMachineIdPrefix = LiteralStringRef("locality_machineid:");
```

Comment:
How about we move these two constants inside struct _LocalityData_?
As I feel rare cases for others to use it except interacting with LocalityData